Cingulara/openrmf-api-read
# release

- updating the version
- updating the README and calls
- updating the version for the container
- Removing old reports, now in the Reports API
- Removing old code
- Adding systemgroup tests
- initial tests for CI/CD
- #30 add better health check
- formatting issues
- adding Jaeger Tracing framework
- Fixing alignment on rows in XLSX export
- Fixing Title
- #42 API for RAR final touches
- #42 API for RAR
- #41 updating the look of the POAM
- #41 fixing the status in the POA&M
- #41 adjusting the way it shows severity
- #41 update the version of the container
- #41 adding CCI Listing lookup for VULNs
- #41 read API severity cells are strings
- #41 read API endpoint for POAM generation XLSX
- #41 read API endpoint for POAM generation XLSX
- #41 read API endpoint for POAM generation XLSX
- #41 read API endpoint for POAM generation XLSX
- #41 read API endpoint for POAM generation XLSX
- #41 read API endpoint for POAM generation XLSX
- #41 read API endpoint for POAM generation XLSX
- cleaning up old code
- add hostname to the system filter
- fixing checklist null issue again
- updating the version for params to export
- Export to the System Filter only
- Web 118 add filtering on the system checklist list
- #40 fix on the VULN record for the API
- #40 adding a report API for vuln listing
- #39 fixing IP address on checklist files
- #39 getting the test plan API endpoint working
- #39 getting the test plan API endpoint working
- #39 getting the test plan API endpoint working
- #39 getting the test plan API endpoint working
- #39 getting the test plan API endpoint working
- adding version to the metadata
- Update issue templates
- #35 print multiple CCI if there
- Fix for reading CCI to export
- #34 reading the hostname, sanitizing the IP
- Updating the README on Jaeger Client
- Jaeger Tracing added
- adding report API for Nessus Report
- updating to next version for reports
- not reviewed color = white, not blue
- adjusting the Nessus Scan for by host
- read API specific prometheus metrics
- #33 initially getting the Prometheus stuff in
- remove the large XML from systems listing info
- #32 fixing nessus hostname and grouping
- #32 fixing column width on headers
- #32 update to the XLSX for generating Patch Data
- #32 fixing the query for final using the temp one
- #32 exporting the XLSX data
- #32 rework the counts and summary listing
- Fixing the Copyright Notice
- #32 actually parsing the Nessus patch file initial
- #32 adding the nessus patch summary endpoint
- added API logging
- updating API documentation
- updating copyright notice missing
- Fixing version number
- added count on systems
- order systems by title
- Naming the NATS connection
- #27 download Nessus file if it is there
- #27 adding a get system record API
- #27 fixing the system title used throughout
- #27 updating to use system group record
- #27 Allow checklists to link to systems records
- Update issue templates
- #25 updating to version 0.10
- #25 adding NATS reconnect and event handlers
- updating the version
- #79 fixing titles to include Tutela
- fixing tab and line breaks in checklists correctly
- cleaning up Makefile
- fixing env to be upper case
- fixing ENVs for upper case
- Calling the compliance via NATS request/reply
- fixing code for RBAC and API to API comms
- close NATS; clean up data we don't need to send
- adding messaging to get score for export XLSX
- update for POSTing and GETting XLSX correctly
- updating info to version 0.8
- removing unused statements
- Cleaning up code for 0.8
- Fixing Role information and Keycloak address
- AuthN and AuthZ working on APIs
- AuthN and AuthZ with Keycloak
